web/src/components/PreLoader/PreLoader.jsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import HashLoader from "react-spinners/HashLoader";
+
+const override = {
+  display: "block",
+  margin: "0 auto",
+  borderColor: "red",
+};
+
+function PreLoader() {
+  return (
+    <div class="flex content-center justify-center h-screen flex-wrap h-48">
+  <div class="w-1/3 p-2">
+    <div class=" text-center p-2">
+    <div className="sweet-loading h-50">
+
+        <HashLoader
+          color="#2b6cb0"
```

Comment:
```suggestion
          color="#1D4ED8"
```

Add the appropriate color to the HashLoader